charts/mentoring/values.yaml

```diff
@@ -0,0 +1,69 @@
+# Default values for mentoring.
+replicaCount: 1
+
+image:
+  repository: 783633885093.dkr.ecr.us-west-2.amazonaws.com/mentoring
+  pullPolicy: IfNotPresent
+  tag: "0.0.1"
+
+fullnameOverride: "mentoring"
+
+serviceAccount:
+  create: true
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: true
+  annotations:
+    kubernetes.io/ingress.class: nginx-mentoring
+  hosts:
+    - host: mentoring.mozilla.com
+      paths:
+        - path: /
+      backend:
+        serviceName: mentoring
+        servicePort: 80
+
+resources:
+  limits:
+    cpu: 500m
+    memory: 1024Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
+
+autoscaling:
+  enabled: true
+  minReplicas: 1
+  maxReplicas: 3
+  targetCPUUtilizationPercentage: 80
+  targetMemoryUtilizationPercentage: 80
+
+configMap:
+  allowedHosts: '"*"'
+
+externalSecrets: false
+
+postgres:
+  enabled: true
+  fullnameOverride: postgres
+  mysqlUser: mentoring
```

Comment:
Why references to mysql under a postgres key?

Reply:
good catch, this is wrong.
Here we should have the corresponding attributes for creating a PSQL user + DB. I didn't catch it during CI because it is actually not used yet.

This will be used as a part of testing the chart and thus the Application. However, right now the application code itself is not yet ready to have PSQL as a DB backend.
I asked for the app to add the options to support a PSQL DB [here](https://github.com/mozilla/mentoring/pull/57#issuecomment-779153928)